simphony_osp/session/session.py

```diff
@@ -46,6 +48,10 @@
 ENTITY = TypeVar("ENTITY", bound=OntologyEntity)
 
 
+RDF_type = RDF.type
+OWL_inverseOf = OWL.inverseOf
```

Comment:
Would it make sense to define all these somewhere globally and then import them?

Reply:
Honestly what would make sense is to send a PR to RDFLib fixing this (for example with a simple `lru_cache` if the namespace does not change).